src/Core/AdminConsole/OrganizationAuth/UpdateOrganizationAuthRequestCommand.cs

```diff
@@ -51,5 +76,64 @@ public async Task UpdateAsync(Guid requestId, Guid userId, bool requestApproved,
                 updatedAuthRequest.RequestIpAddress, deviceTypeAndIdentifier);
         }
     }
-}
 
+    public async Task UpdateAsync(Guid organizationId, IEnumerable<OrganizationAuthRequestUpdate> authRequestUpdates)
+    {
+        await new BatchAuthRequestUpdateProcessor<OrganizationAdminAuthRequest>(
+            await FetchManyOrganizationAuthRequestsFromTheDatabase(organizationId, authRequestUpdates.Select(aru => aru.Id)),
+            authRequestUpdates,
+            new AuthRequestUpdateProcessorConfiguration()
+            {
+                OrganizationId = organizationId,
+                AuthRequestExpiresAfter = _globalSettings.PasswordlessAuth.AdminRequestExpiration
+            }
+        )
+        .Process((Exception e) => _logger.LogError(e.Message))
```

Comment:
The try/catch could be in this method and you wouldn't need to pass the error action as a parameter

Reply:
This `try / catch` needs to go directly inside whatever loop runs the approve/deny logic for each update so it can handle multiple updates being invalid without halting the loop.